package.json

```diff
@@ -58,12 +58,15 @@
     "test.prod": "yarn workspace @stoplight/elements-utils test.prod && yarn workspace @stoplight/elements test.prod && yarn workspace @stoplight/elements-core test.prod && yarn workspace @stoplight/elements-dev-portal test.prod --passWithNoTests",
     "prepublishOnly": "yarn build",
     "/// examples ///": "",
-    "build:angular": "(cd ./examples/angular && yarn reinstall && yarn build)",
-    "serve:angular": "(cd ./examples/angular && yarn serve)",
-    "build:react-gatsby": "(cd ./examples/react-gatsby && yarn reinstall && yarn build)",
-    "serve:react-gatsby": "(cd ./examples/react-gatsby && yarn serve)",
-    "build:react-cra": "(cd ./examples/react-cra && yarn reinstall && yarn build)",
-    "serve:react-cra": "(cd ./examples/react-cra && yarn serve)",
+    "copy:angular": "mkdir examples-dev ; cp -a -v ./examples/angular ./examples-dev/angular ; . ./use-local-elements.sh angular",
+    "build:angular": "(cd ./examples-dev/angular && yarn reinstall && yarn build)",
+    "serve:angular": "(cd ./examples-dev/angular && yarn serve)",
+    "copy:react-gatsby": "mkdir examples-dev ; cp -a ./examples/react-gatsby ./examples-dev/react-gatsby ; . ./use-local-elements.sh react-gatsby",
+    "build:react-gatsby": "(cd ./examples-dev/react-gatsby && yarn reinstall && yarn build)",
+    "serve:react-gatsby": "(cd ./examples-dev/react-gatsby && yarn serve)",
+    "copy:react-cra": "mkdir examples-dev ; cp -a ./examples/react-cra ./examples-dev/react-cra ; . ./use-local-elements.sh react-cra",
```

Comment:
Might make sense to first remove already existing `examples-dev` directory to have it always up to date

Reply:
Fortunately `cp` overwrites files by default. Or do you mean something else?

Reply:
hmm, it doesn't for me:
```
mkdir examples-dev ; cp -a ./examples/react-cra ./examples-dev/react-cra ; . ./use-local-elements.sh react-cra
mkdir: cannot create directory ‘examples-dev’: File exists
```

Reply:
Right it does now.
One last thing I'm wondering is if it's worth to include copying with build:example command. It would run with every build but don't we need it to behave this way to keep everything up to date?